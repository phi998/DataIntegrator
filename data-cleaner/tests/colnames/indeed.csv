Advertisement Content,Advertisement Title,Company Name,Advertisement Content,Advertisement Content,Advertisement Title,Advertisement Title
