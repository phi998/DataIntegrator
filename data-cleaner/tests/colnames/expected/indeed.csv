Advertisement Title,Advertisement Title,Company Name,Advertisement Content,.,Company Name,.
